br/pkg/restore/client.go

```diff
@@ -1348,6 +1354,7 @@ func (rc *Client) IsSkipCreateSQL() bool {
 // 2. create table/create and rename table: the first create table will lock down the id cache.
 // because we cannot create onExistReplace table.
 // so the final create DDL with the correct auto increment/random id won't be executed.
+// Deprecated
```

Comment:
we can remove this function directly, since there no other caller depend on this function.